Our Scripts/MPhys_model.py

```diff
@@ -126,14 +124,18 @@ def EW(z):
 
     available at: https://lancaster.app.box.com/s/t75t3v713yuibkvjk3ioqdesunxpv1fb/file/618125008170
     """
-    x = pylab.array([2.5,2.8,2.9,3.1, 3.3, 3.7, 4.1, 4.5, 4.8, 5.0, 5.3])
-    y = pylab.array([117.134349876, 122.113769531, 172.679885864, 143.5936203, 149.371124268, 96.3648490905, 189.002449036, 181.127731323, 95.0448436864, 125.935153962, 143.343048096])
-    y_sigma = pylab.array([0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01, 0.01, 0.01])
+    x = pylab.array([2.5,2.8,2.9,3.1, 3.3, 3.7, 4.1, 4.5, 4.8, 5.0, 5.3, 5.8])
+    y = np.array([117.134349876, 122.113769531, 172.679885864, 143.5936203, 149.371124268, 96.3648490905, 189.002449036, 181.127731323, 95.0448436864, 125.935153962, 143.343048096, 251.93561])
+    y_err_up =pylab.array([186.35171, 160.96419, 321.31418, 382.54525, 258.18398, 70.23115, 763.57573, 223.72103, 100.98794, 195.63354, 159.8391, 620.82504])
+    y_err_down =pylab.array([53.82247, 53.55149, 90.78461, 72.25881, 74.65443, 39.08451, 96.45726, 85.27828, 42.09321, 52.08989, 71.00167, 186.34499])
+    error = [(y_err_up[i]-y_err_down[i])/2 for i in range(len(y_err_down))]
     p2 = np.polyfit(x, y, 1.0)
     p = pylab.poly1d(p2)
     Params = p.coefficients
-    pfit, pcov = curve_fit(linear, x, y, p0=Params, sigma=y_sigma)
+    pfit, pcov = curve_fit(linear, x, y, p0=Params, sigma=error)
     a1,a2= pfit[0], pfit[1]
+    aver = np.mean(y)
+    print(aver)
```

Comment:
Just double checking that this is more up to date

Reply:
yep, that is the one 😊